nengo_viz/disposable_js.py

```diff
@@ -35,6 +35,12 @@ def configmodal():
     js.append(show_modal_js())
     return '\n'.join(js)
 
+def helpmodal():
+    js = [add_modal_title_js('Hotkeys list')]
```

Comment:
yeah i was just thinking about this last night, meant to do it then. i'll do it now!